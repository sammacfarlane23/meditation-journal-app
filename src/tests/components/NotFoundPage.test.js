import React from 'react';
import NotFoundPage from '../../components/NotFoundPage';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';

test('should render NotFoundPage correctly', () => {
  const component = render(
    <Router>
      <NotFoundPage />
    </Router>
  );
  expect(component).toMatchSnapshot();
});
