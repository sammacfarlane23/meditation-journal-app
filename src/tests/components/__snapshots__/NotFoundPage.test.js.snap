// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page correctly 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/dashboard"
  >
    Go home
  </Link>
</div>
`;
